.circleci/config.yml

```diff
@@ -365,6 +365,26 @@ jobs:
       - save_cache:
           key: manylinux2010-x64-assets-{{ .Revision }}
           paths: ~/docker/manylinux2010-x64.tar
+  manylinux2010-x86:
```

Comment:
We also need to enable at the end of the *config.yml* file.

Reply:
Done! Thanks!